src/mmrelay/db_utils.py

```
@@ -253,48 +259,46 @@ def get_db_path() -> str:
-            if path and os.path.exists(path)
-        ]
-        default_path = _migrate_legacy_db_if_needed(
-            default_path=default_path,
-            legacy_candidates=legacy_candidates,
-        )
+    default_path = os.path.join(database_dir, "meshtastic.sqlite")
+
+    # Check for legacy database in legacy sources and migrate if needed
+    # Also check if custom home directory was specified (MMRELAY_HOME/--home)
+    home_source = paths_info.get("home_source")
+    new_layout_enabled = is_new_layout_enabled() or home_source in {
```

Ruff (0.14.14)

[error] Local variable `new_layout_enabled` is assigned to but never used

Remove assignment to unused variable `new_layout_enabled`

(F841)